editor/i18n/zh/assets.js

```diff
@@ -16,6 +16,11 @@ module.exports = {
             animation: '动画',
             loop: '循环',
             timeScale: '速率',
+            premultipliedAlpha: 'Premultiplied Alpha',
+            useTint: 'Use Tint',
+            debugSlots: 'Debug Slots',
+            debugBones: 'Debug Bones',
+            debugMesh: 'Debug Mesh',
```

Comment:
Why these are english, and the above, such as `loop` is Chinese?

Reply:
These are more intuitive in English, or we can temporarily use all English for now and replace them with Chinese property names later.

Reply:
I think we should do it properly currently.

Reply:
done